app/services/scrapers/judges.rb

```diff
@@ -17,7 +17,7 @@ def perform
       # rubocop:enable Layout/LineLength
 
       html = HTTParty.get(base_url, headers: {
-                            'User-Agent': user_agent,
+                            'User-Agent': ENV.fetch('USER_AGENT', user_agent),
```

Comment:
instead of giving  User-Agent a value from a regular variable, you simply gave it a value from an environment variable.My guess as to why is to have the value be tied closer to the  user's session?